src/contributors/VaasuCodez.json

```
@@ -0,0 +1,5 @@
+{
+    "name": "Vaasu", 
+    "desc": "Student",
+    "github": "https://github.com/VaasuCodez",
```

```suggestion
    "github": "https://github.com/VaasuCodez"
```
Remove this trailing comma or else add your twitter or linkedin profile.

---

Thanks a bunch!